README.md

```diff
@@ -1,2 +1,26 @@
-# hw01
-Repository for first CFSS homework assignment
+## Some facts about me
+* **Name**: Soo Wan Kim
+* **Born**: Seoul, South Korea
+* **Current residence**: Chicago, IL
+
+## Education/Career
+* **Undergraduate**: University of Chicago Class of 2014, Economics
+* **Current**: [MAPSS](https://mapss.uchicago.edu/), focusing on international relations and computational social science
+* **Research interests**: international security, terrorism
+
+## Timeline
+##### I haven't lived that long.
+* *Summer 2010*, Albany NY - graduated high school
+* *Fall 2010* - started college
+* *Spring 2014* - first RA job
+* *Summer 2014* - graduated college
+* *Fall 2016* - started master's program
+* *Winter 2017* - enrolled in Computing for the Social Sciences
+
+![](https://m.popkey.co/4ab02f/6gMO_f-maxage-0.gif)
+
+## Making this page
+Setting up everything on Git and RStudio was easy since I had used them previously. 
+I used the guide on [this page](http://cfss.uchicago.edu/git05.html) to clone the GitHub repository for homework 1 via RStudio.
+For using Markdown, [this GitHub guide](https://guides.github.com/features/mastering-markdown/) was helpful.
+At first I couldn't figure out how to preview my Markdown file, but then I followed the instructions on [this page](http://rmarkdown.rstudio.com/lesson-1.html) to install the R Markdown package, which opened up the "Preview HTML" option in RStudio.
```

Comment:
In the future, this type of feedback should go in the pull request comment, not the assignment repo itself. But I'm glad you found some useful resources to resolve your problem!